src/tools/genpolicy/src/registry.rs

```diff
@@ -214,48 +272,43 @@ impl Container {
                         }
 
                         debug!("Parsing gid from user[1] = {:?}", user[1]);
-                        match user[1].parse() {
-                            Ok(id) => process.User.GID = id,
+                        match user[1].parse::<u32>() {
+                            Ok(id) => {
+                                warn!(
+                                    concat!(
+                                        "Parsed gid {} from OCI container image config, but not using it. ",
```

Comment:
Is this CRI-dependent? If so, this is not the guaranteed behavior, potentially rephrase